src/jira/client/jira-api-client.ts

```diff
@@ -0,0 +1,494 @@
+import Logger from "bunyan";
+import { getAxiosInstance, JiraClientError } from "./axios";
+import { AxiosInstance, AxiosResponse } from "axios";
+import { Installation } from "models/installation";
+import { Subscription } from "models/subscription";
+import { envVars } from "config/env";
+import { uniq } from "lodash";
+import { jiraIssueKeyParser } from "utils/jira-utils";
+import { TransformedRepositoryId, transformRepositoryId } from "~/src/transforms/transform-repository-id";
+import { getJiraId } from "../util/id";
+import { getCloudOrServerFromGitHubAppId } from "utils/get-cloud-or-server";
+import { getDeploymentDebugInfo, extractDeploymentDataForLoggingPurpose } from "./jira-client-deployment-helper";
+import {
```

Comment:
all these functions used to exists  here and there around the outside of the classes, i have bundled them into a helper file... more on that when we get to it